vitals/Main.java

```diff
@@ -1,23 +1,44 @@
 package vitals;
+import java.util.logging.Logger;
 
 public class Main {
-    static boolean batteryIsOk(float temperature, float soc, float chargeRate) {
-        if(temperature < 0 || temperature > 45) {
-            System.out.println("Temperature is out of range!");
-            return false;
-        } else if(soc < 20 || soc > 80) {
-            System.out.println("State of Charge is out of range!");
-            return false;
-        } else if(chargeRate > 0.8) {
-            System.out.println("Charge Rate is out of range!");
-            return false;
+     static boolean batteryIsOk(float temperature, float soc, float chargeRate) {
```

Comment:
it appears that all the parameter check function repeats if/else range check ! DRY principle violated?